task.go

```diff
@@ -208,7 +208,7 @@ func (r *marathonClient) TaskEndpoints(name string, port int, healthCheck bool)
 	return list, nil
 }
 
-func (r *Task) allHealthChecksAlive() bool {
+func (r *Task) AllHealthChecksAlive() bool {
```

Comment:
Fine with me. We should then

- comment the newly exported method, explaining when it returns true and false, respectively; and 
- make sure we cover all code paths through a test. (Can't do this right now while being on the phone, unfortunately.)